entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+set -e
+
+NB_POSTGRESQL_DB=pfb
+NB_POSTGRESQL_USER=gis
+NB_POSTGRESQL_PASSWORD=gis
+
+# start postgres and capture the PID
+/docker-entrypoint.sh postgres &
+POSTGRES_PROC=$!
+
+# sleep while db comes up
+sleep 10
+
+# set up database
+su postgres bash -c psql <<EOF
```

Comment:
Addressed in https://github.com/azavea/pfb/pull/15/commits/63aeaf1f26162a38d0f18987670e4f6e88f32a59